.travis.yml

```diff
@@ -3,14 +3,18 @@ language: node_js
 node_js:
   # we recommend testing addons with the same minimum supported node version as Ember CLI
   # so that your addon works for all apps
-  - "4"
+  - "6"
```

Comment:
can we leave it at version 4 if it passes CI still?

Reply:
node 4 lets ember test pass, but it does not work with your current mocha test.  If you want to skip your mocha test then this can go back to 4.

Reply:
ignore. saw comment below.

Reply:
You can see the error here:  https://travis-ci.org/skaterdav85/ember-cli-doorkeeper/jobs/355560405

Reply:
@skaterdav85 have you tried running your tests with node@4 locally?